onnxruntime/core/providers/cpu/nn/pool_attributes.h

```diff
@@ -90,7 +90,7 @@ struct PoolAttributes {
   std::vector<int64_t> SetOutputSize(const TensorShape& input_shape,
                                      int64_t output_channel,
                                      std::vector<int64_t>* actual_pads) const {
-    ORT_ENFORCE(input_shape.Size() > 0);
+    ORT_ENFORCE(input_shape.Size() >= 0);
```

Comment:
What happens when input size is 0? No early exit?

Reply:
Correct. It still need to calculate the output dims so that the shape for the output is correct.